docs/source/trustme-trio-example.py

```diff
@@ -15,6 +19,13 @@
     # Set up the server's SSLContext to use our fake server cert
     server_cert.configure_cert(server_ssl_context)
 
+    # Set up the server's SSLContext to trust our fake CA, that signed
+    # our client cert
+    client_ca.configure_trust(server_ssl_context)
+
+    # Verify that client sent us their TLS cert signed by a trusted CA
+    server_ssl_context.verify_mode = ssl.CERT_REQUIRED
```

Comment:
@njsmith here it forces the server to do CA trust validation so that import above is required

Reply:
We're using `trio.ssl` everywhere else, so I guess we should probably use it here too for consistency. (`trio.ssl` re-exports all the `ssl` constants.)

Reply:
Oh... I checked trio's source and probably overlooked that it's imported then.

Reply:
I see now, it's there.